apps/studio/components/interfaces/Integrations/Landing/Integrations.constants.tsx

```diff
@@ -173,6 +173,7 @@ const supabaseIntegrations: IntegrationDefinition[] = [
     id: 'vault',
     type: 'postgres_extension' as const,
     requiredExtensions: ['supabase_vault'],
+    missingExtensionsAlert: <UpgradeDatabaseAlert />,
```

Comment:
This component is specific to Queues. It says `The extension is available on version 15.6.1.143 and higher.`. Technically, it's correct (you can upgrade to a lower version than that) but the extension is available since PG v15 (I think).

Reply:
Should we add another field for minimum version and use that in the alert?

Reply:
> should we add another field for minimum version

that makes sense! although it wouldn't matter too much atm since we don't support selecting a specific version to upgrade to (upgrades are always just the latest version iinw)

i'll still add a minimum version prop nonetheless, and default it to 15.6 for now

Reply:
okay pushed :)